packages/angular/src/lib/charts/wordcloud-chart.component.ts

```diff
@@ -20,9 +16,9 @@ export class WordCloudChartComponent extends BaseChartComponent implements After
 	 * Runs after view init to create a chart, attach it to `elementRef` and draw it.
 	 */
 	override ngAfterViewInit() {
-		this.chart = new WordCloudChartCore(this.elementRef.nativeElement, {
-			data: this.data as ChartTabularData,
-			options: this.options as WorldCloudChartOptions
+		this.chart = new WordCloudChartCore(this.elementRef.nativeElement as HTMLDivElement, {
+			data: this.data,
+			options: this.options as WordCloudChartOptions
```

Comment:
should we do WordCloudChartOptions | WorldCloudChartOptions? typescript could still throw an error not matching the type names

Reply:
Got this updated.